code/datums/vote/vote.dm

```diff
@@ -47,13 +51,14 @@
 
 /datum/vote/proc/start_vote()
 	start_time = world.time
+	time_set = (time_set ? time_set : config.vote_period) SECONDS
```

Comment:
Confusing; this means that at compile time this var is in seconds, but then is multiplied by 10 at runtime?